functions-methods.js

```diff
@@ -19,8 +21,17 @@
 // typeOfEmail("t.mellink@novi.nl") geeft geeft "Medewerker"
 // typeOfEmail("novi.nlaapjesk@outlook.com") geeft geeft "Extern" <-- deze moet het ook doen!
 // typeOfEmail("a.wiersma@outlook.com") geeft "Extern"
+function typeOfEmails(emailAddress) {
+    if (emailAddress.split("@")[1].includes("novi-education.nl")) {
+        return "Student";
+    } else if (emailAddress.split("@")[1].includes("novi.nl")) {
```

Comment:
Als je dezelfde code vaker gaat gebruiken kun je die beter in een variabelen zetten:

const domain = emailAddress.split("@")[1]